File: Taller Tipos de clases/Go/Clase Normal/Estudiante.go
class Estudiante
{
    public string Id;
    public int Semestre;

    public Estudiante()
    {
        Id = "000";
        Semestre = 1;
    }

    public Estudiante(string id, int semestre)
    {
        Id = id;
        Semestre = semestre;
    }

    public Estudiante(string id)
    {
        Id = id;
        Semestre = 1;
    }

    public Estudiante(Estudiante e)
    {
        Id = e.Id;
        Semestre = e.Semestre;
    }
}

File: Taller Tipos de clases/JAVA/Clase Normal/Estudiante.java
class Estudiante {
    String id;
    int semestre;

    public Estudiante() {
        id = "000";
        semestre = 1;
    }

    public Estudiante(String id, int semestre) {
        this.id = id;
        this.semestre = semestre;
    }

    public Estudiante(String id) {
        this.id = id;
        this.semestre = 1;
    }

    public Estudiante(Estudiante e) {
        this.id = e.id;
        this.semestre = e.semestre;
    }
}
